warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -2089,6 +2089,7 @@ protected Future<IteratorSetting> loadQueryIterator(final MetadataHelper metadat
             addOption(cfg, QueryOptions.MAX_EVALUATION_PIPELINES, Integer.toString(config.getMaxEvaluationPipelines()), false);
             addOption(cfg, QueryOptions.MAX_PIPELINE_CACHED_RESULTS, Integer.toString(config.getMaxPipelineCachedResults()), false);
             addOption(cfg, QueryOptions.MAX_IVARATOR_SOURCES, Integer.toString(config.getMaxIvaratorSources()), false);
+            addOption(cfg, QueryOptions.MAX_IVARATOR_SOURCE_WAIT, Long.toString(config.getMaxIvaratorSourceWait()), false);
```

Comment:
Since the default value is always 30 minutes in the ShardQueryConfig, the QueryOptions, and the Ivarator we only need to add the option if it is a non-default value.

Reply:
That would need to be done for all of the properties.  I think I will leave that to a separate PR.